bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
```

Comment:
can you use imports? That would be nice because it will activate import linting.